### ISSUE TITLE
A11y_AzureDataStudio_Welcome_Resources_ScreenReader : The "History" control is not announced with proper role for screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/azuredatastudio#11547]
**"[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”**
GitHubTags:#A11y_AzureDataStudioJuly2020;#A11y_July27_2020_TestPass;#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#MAC;#Screenreader;#VoiceOver;#A11ySev2;#Benchmark;#MAS1.3.1;#MAS4.1.2;#FTP;

### Environment Details:
Application Name: Azure Data Studio
Application Version: 1.21.0-insider
Commit: eccf3cf
Date: 2020-07-24T09:28:31.172Z
VS Code: 1.48.0
Electron: 9.1.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina (Version 10.15.6 (19G73)
Screen Reader: VoiceOver
MAS References: MAS1.3.1, MAS4.1.2

### Repro Steps:

1. Launch Azure Data Studio Insiders application.
2. If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3. Navigate till Resources section on Welcome screen.
4. Start screen reader, navigate to the "History" control and listen if it is announced as 'group' or 'text'.

### Actual:
When screen reader users navigate to the "History" control it is announced as group and text which is incorrect as per the visual appearance of the control.

### Expected:
The "History" control should be announced with role as 'Tab' for screen reader users.

### User Impact:
If proper role of the interactive element is not announced to the screen reader users then they will not know how to interact with that element.

### Attachment link for Reference:
[11547_A11y_AzureDataStudio_Welcome_Resources_ScreenReader The "History" control is not announced with proper role for screen reader users.zip](https://github.com/microsoft/azuredatastudio/files/4985800/11547_A11y_AzureDataStudio_Welcome_Resources_ScreenReader.The.History.control.is.not.announced.with.proper.role.for.screen.reader.users.zip)

**Comment by Harika1702** [comment]
<img width="1440" alt="11547_A11y_fixed" src="https://user-images.githubusercontent.com/100270074/218695323-fcc0f609-53e3-4659-8402-95ec309156ba.png">
Verified the issue Now it is Fixed.

Git Hub Tags: #Closed;
